course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
yup.  i think you can do that by chaining a .then() on the end of the get, and putting the call inside the failure callback on the post.